Lab3.ipynb

```diff
@@ -0,0 +1,366 @@
+   "metadata": {},
+   "source": [
+    "From the code, we can see how there is a Gaussian distribution with 100k 'good' data points where mean is 12 and sigma is 0.4. This is because we want the temperature to be around 12 Kelvin with 0.4 degree precision of the thermometry. There are also 'bad' data points to represent error in the thermometry or temperature readings. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Section 1A.2**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The statistical question I have is how likely is the temperature reading to be produced by the background? The reason this is my statistical question is because the experiment works reliably when the temperature is around 12 Kelvin. According to the data, there is a 0.4 degree of precision of thermometry. We want to remove any data points which exceed the statistical threshold. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Section 1A.3**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "My statistical question can be rephrased mathematically. Here, we know we have a Gaussian distribution of 100k 'good' data points where mean is 12 and sigma is 0.4. There are also 'bad' data points to represent error in thermometry. We want to filter out any data point which are n-sigma away from the mean. Mathematically, this means:\n",
+    "\n",
+    "$\\sigma = \\frac{|X - \\mu|}{0.4}$ "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Section 1A.4**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "d = np.array(d)\n",
+    "sigma = abs(d-12)/0.4\n",
+    "x = {'Temperature' : d, 'Sigma' : sigma}\n",
+    "temp = pd.DataFrame(x)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "goodtemp = temp[temp['Sigma'] < 4]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "badtemp = temp[temp['Sigma'] > 4]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Section 1B**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The number of omissions depends on the statistical threshold we chose. This is because it is a predictable quantity since we know how much of the good data we threw out. Specifically, a z-table shows 1-sigma corresponds to ~68% of the data, 2-sigma correponds to ~95% of the data, and 3-sigma corresponds to ~99% of the data. This means approximately 32%, 5%, and 1% of the data is thrown out respectively. This can be applied to other sigma values using the z-table where the percentage thrown out is simply subtracting 100 by the value corresponding to n-sigma. "
```

Comment:
Should it be 100% minus the percentage given by the z table? that are 1 minus the probability given by the table. The end is just ambiguous about if you are talking about percentage point although it is very clear from context what you mean